src/EditorFeatures/CSharpTest/AddAccessibilityModifiers/AddAccessibilityModifiersTests.cs

```diff
@@ -310,7 +310,7 @@ enum E
                 },
                 Options =
                 {
-                    { CodeStyleOptions.RequireAccessibilityModifiers, AccessibilityModifiersRequired.OmitIfDefault },
```

Comment:
This is a _big commit_, but it is a test-only commit with almost all edits being simple find and replace and absolutely no product code changes.